PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebCheckoutClient.kt

```diff
@@ -119,4 +133,13 @@ class PayPalWebCheckoutClient internal constructor(
         analyticsService.sendAnalyticsEvent("paypal-web-payments:vault-wo-purchase:canceled")
         vaultListener?.onPayPalWebVaultCanceled()
     }
+
+    /**
+     * Call this method at the end of the web checkout flow to clear out all observers and listeners
+     */
+    fun removeObservers() {
+        activityReference.get()?.let { it.lifecycle.removeObserver(observer) }
```

Comment:
is lifecycle always non null here?

Reply:
My guess would be that it's always not null, but I don't know that much about Android. Perhaps I can add some guards around it